livereveal/main.js

```diff
@@ -17,8 +17,14 @@ var config_section = new configmod.ConfigSection('livereveal',
                             {base_url: utils.get_body_data("baseUrl")});
 config_section.load();
 var config = new configmod.ConfigWithDefaults(config_section, {
+    controls: true,
+    progress: true,
+    history: true,
+    width: 1140,
+    minScale: 1.0,
```

Comment:
Wouldn't it be enough to document that setting it to another value will break codemirror?  This way, those who do not use codemirror can still play with this setting.

As far as I am concerned, I have no usage for codemirror in presentation mode (I hide all code cells and do all the interactivity via widgets), but I've had many trouble to adapt my presentation to the width of the beamer (that I cannot know before hand) because of scaling issues.